test/Sleep-test.js

```diff
@@ -47,27 +47,27 @@ describe('Sleep', () => {
   });
 
   it('should return the average sleep hours for a single user over all time', () => {
-    expect(sleep.returnAvgSleepHours()).to.equal(7.66);
+    expect(sleep.returnAvgSleepInfo('hoursSlept')).to.equal(7.66);
   });
 
   it('should return the average sleep quality over all time for a single user', () => {
-    expect(sleep.returnAvgSleepQuality()).to.equal(2.53);
+    expect(sleep.returnAvgSleepInfo('sleepQuality')).to.equal(2.53);
   });
 
   it('should return how many hours slept for a specific day', () => {
-    expect(sleep.returnSleepHours('2019/06/15')).to.equal(6.1);
+    expect(sleep.returnDaysSleepInfo('2019/06/15', 'hoursSlept')).to.equal(6.1);
   });
 
   it('should return sleep quality for a specific day', () => {
-    expect(sleep.returnSleepQuality('2019/06/15')).to.equal(2.2);
+    expect(sleep.returnDaysSleepInfo('2019/06/15', 'sleepQuality')).to.equal(2.2);
   });
 
   it('should return hours slept each day for week for a specific user', () => {
-    expect(fullSleep.returnWeekOfSleepHours(2)).to.eql([7.3, 5.1, 8.6, 10.5, 9.1, 6.5, 6.8]);
+    expect(fullSleep.returnWeekOfSleepInfo(2, 'hoursSlept')).to.eql([7.3, 5.1, 8.6, 10.5, 9.1, 6.5, 6.8]);
   });
 
   it('should return hours slept each day for week for a specific user', () => {
```

Comment:
- ('should return hours slept each day for week for a specific user')
The should statement here should refer to sleep quality rather than hours slept